mongo/mongo.go

```diff
@@ -0,0 +1,94 @@
+package mongo
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dpMongoLock "github.com/ONSdigital/dp-mongodb/v2/dplock"
+	dpMongoHealth "github.com/ONSdigital/dp-mongodb/v2/health"
+	dpmongo "github.com/ONSdigital/dp-mongodb/v2/mongodb"
+)
+
+const (
+	connectTimeoutInSeconds = 5
+	queryTimeoutInSeconds   = 15
+
+	editionsCollection     = "editions"
+	instanceCollection     = "instances"
+	instanceLockCollection = "instances_locks"
+	dimensionOptions       = "dimension.options"
+)
+
+// Mongo represents a simplistic MongoDB configuration.
+type Mongo struct {
+	Username     string
+	Password     string
+	IsSSL        bool
+	CodeListURL  string
+	Collection   string
+	Database     string
+	DatasetURL   string
+	Connection   *dpmongo.MongoConnection
+	URI          string
+	client       *dpMongoHealth.Client
+	healthClient *dpMongoHealth.CheckMongoClient
+	lockClient   *dpMongoLock.Lock
+}
+
+func (m *Mongo) Init(ctx context.Context) (err error) {
+	if m.Connection != nil {
+		return errors.New("mongo connection already exists")
+	}
+
+	connCfg := &dpmongo.MongoConnectionConfig{
+		IsSSL:                   m.IsSSL,
+		ConnectTimeoutInSeconds: connectTimeoutInSeconds,
+		QueryTimeoutInSeconds:   queryTimeoutInSeconds,
+
+		Username:                      m.Username,
+		Password:                      m.Password,
+		ClusterEndpoint:               m.URI,
+		Database:                      m.Database,
+		Collection:                    m.Collection,
+		IsWriteConcernMajorityEnabled: false,
+		IsStrongReadConcernEnabled:    true,
```

Comment:
`IsStrongReadConcernEnabled` should be set to `false` and `IsWriteConcernMajorityEnabled` should be set to `true`